homeassistant/components/bmw_connected_drive/coordinator.py

```diff
@@ -27,34 +28,48 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+type BMWConfigEntry = ConfigEntry[BMWData]
+
+
+@dataclass
+class BMWData:
+    """Class to store BMW runtime data."""
+
+    coordinator: BMWDataUpdateCoordinator
```

Comment:
If you're anyway going to move the typing I suggest you drop `BMWdata` and attach the coordinator directly to runtime. No need to have a data class when there is only one thing in it.

Reply:
Was not sure if it was needed at all, but the original blog post mentioned it. However happy to remove the extra step.